templates/doc-sections/creating-your-own-workshop.phtml

```diff
@@ -0,0 +1,62 @@
+<h2 id="creating-your-own-workshop" class="doc__title">Creating your own workshop<a class="anchor" href="#creating-your-own-workshop">#</a><a href="#page-top" class="back-to-top">^ TOP</a></h2>
+<p>So.. You want to create your own workshop? You want to help teach a topic? You want to make a meetup more hands on? You want to showcase your tool? You've come to the right place!</p>
+
+<p>In order to create a workshop the first thing you will need to do is to setup a new project and configure the dependencies and wire it all together. Luckily for you we have created a starter kit which does this all for you, prompting you along the way. Simply execute the following command in your shell.</p>
+
+<div class="terminal">
+    <div class="terminal-head">
+        <span class="terminal__button"></span>
+        <span class="terminal__button"></span>
+        <span class="terminal__button"></span>
+    </div>
+    <div class="terminal__body">
+        <p>composer create-project php-school/workshop-starter-kit &lt;your-workshop-name&gt;</p>
+    </div>
+</div>
+
+<p>Where <code>&lt;your-workshop-name&gt;</code> is the name of the directory you want to setup your workshop in. For this tutorial we will create a workshop teaching some basic math concepts so we will use the name <code>simple-math</code>. You will be prompted for a few things:</p>
+
+<h3>Composer package name</h3>
+<p>The name for your workshop to be put in the <code>composer.json</code> file. This should follow the pattern <code>vendor/package</code>. In the case of our flagship workshop <a target="_blank" href="https://github.com/php-school/learn-you-php">Learn You PHP!</a> this is <code>php-school/learn-you-php</code>. Name your package <code>php-school/simple-math</code>.</p>
+<p>More info on the <code>name</code> property can be found <a target="_blank" href="https://getcomposer.org/doc/04-schema.md#name">here</a>.</p>
```

Comment:
well maybe anyway, unless it reads weird because there's a few links to composer in this section